pykilosort/gui/data_view_box.py

```diff
@@ -337,14 +351,20 @@ def change_displayed_channel_count(self, direction):
         if 0 < new_count <= total_channels:
             self.set_currently_displayed_channel_count(new_count)
             self.channelChanged.emit()
+            self.plot_item.clear()
+            self.create_plot_items()
             self.update_plot()
```

Comment:
we repeat these 4 lines together, do they have a name as a single operation? like
```
class DataViewBox
    ...
    def refresh(self):
        self.channelChanged.emit()
        self.plot_item.clear()
        self.create_plot_items()
        self.update_plot()
```

Reply:
good point
made a new function from them!